src/svelte/fetcher.ts

```diff
@@ -14,7 +14,28 @@ import type {
   Method,
 } from '../types.js'
 import type { ApiRequest, ApiResponse, SvelteCreateFetch } from './types.js'
-import { ApiError } from '../types.js'
+import { ApiError, LimitedResponse } from '../types.js'
+
+async function getResponseBody(response: LimitedResponse) {
+  // no content
+  if (response.status === 204) {
+    return undefined
+  }
+  const contentType = response.headers.get('content-type')
+  if (contentType && contentType.indexOf('application/json') !== -1) {
+    return await response.json()
```

Comment:
Please add make a list of valid JSON mime types to check against, and add `application/ld+json` at a minimum
```suggestion
  const jsonTypes = ["application/json", "application/ld+json"]
  if (contentType && jsonTypes.includes(contentType)) {
    return await response.json()
```